README.md

```diff
@@ -30,6 +30,28 @@ $ find . -name '*.go' -not -path './vendor*' -exec go fmt {} \;
 
 > Environment variables https://github.com/mdouchement/breizhtrip-go/blob/master/config/config.go
 
+## Import data to database
+
+You need to have a database name "breizhtrip_development".
```

Comment:
- or `breizhtrip_production` or `breizhtrip_test` according to the Gin Gonic mode.